### ISSUE TITLE
Feature request: more to :<unit> from :string methods

### ISSUE THREAD

**Issue by Ryton** [arturo-lang/arturo#1343]
**Describe the bug**
           | cannot convert argument: "1.0+0.0i"
           | from :string
           | to   :complex

**To Reproduce**
complStr: to :string to :complex [1 0]
complexNr: to :complex a ; <= error: cannot convert nr.

**Expected behavior**
b: [1 0] 

*** possible workaround ** (but not yet supporting neg values and more complex parsings...

_getvalofcomplexstring: $[complstr][   to :complex [first split.by: "+" complStr chop last split.by: "+" complStr] ]_

I could parse it w regex too but would/could/should be usefull to include in the to :string function as well (once robustified?

**Comment by Ryton** [comment]
ah, though that string to float" was exisiting already => but it is not.
=> more like a feature reques than  a bug repor then:

print to :float "1.23" ; => | cannot convert argument: "1.23"

**Comment by drkameleon** [comment]
@Ryton I might, just *might*, have an idea how you came up with issue... 🤣 (thought of doing it with complex numbers as well, but then decided to simply use blocks)

Kidding aside, thanks for mentioning *and* opening an issue! 🚀 
(that's by far the best way to keep track of everything and not simply forget about it...)

**Comment by Ryton** [comment]
yes, thats why indeed ^^ This approach worked well on AoC #10 so wanted to rehash it. 

but unfortunately it isnt as straightforward as i thought it would be. but since i do have the source numbers, i can just pass them along as a list too. A bit more cumbersome but  doable too.
